boot/BootFromDevice.c

```diff
@@ -271,6 +298,8 @@ int BootFromDevice(CONFIGENTRY *config) {
 	if (result) {
 		if (config->bootSystem == SYS_LINUX)
 			ExittoLinux(&config->opt.Linux);
+		if (config->bootSystem == SYS_REACTOS)
+			ExittoReactOS(&config->opt.Multiboot);
```

Comment:
nit: It's weird how we now have `bootSystem` as `REACTOS`, but use `opt.Multiboot` (`OPTMULTIBOOT`).

(Not sure if I had requested this, but this doesn't make much sense - it should be made consistent)

Reply:
The structure itself doesn't contain any ReactOS specific values, that's why I called it like that. Also it gives an idea for future refactoring, as discussed in https://github.com/XboxDev/cromwell/pull/27#discussion_r363062315.